google-beta/resource_container_cluster.go

```diff
@@ -2721,7 +2721,10 @@ func flattenMasterAuth(ma *containerBeta.MasterAuth) []map[string]interface{} {
 
 func flattenClusterAutoscaling(a *containerBeta.ClusterAutoscaling) []map[string]interface{} {
 	r := make(map[string]interface{})
-	if a == nil || !a.EnableNodeAutoprovisioning {
+	if a == nil {
+		r["enabled"] = false
+		return []map[string]interface{}{r}
+	} else if !a.EnableNodeAutoprovisioning {
```

Comment:
This is minor, but to stay consistent with some of our coding practices, I would probably start a new `if` block here, rather than having this an `else if` since the previous block will be returning anyway.

Also minor, but I personally find it easier to read an `if true` block, rather than `if false`.

Reply:
You're right. ;) It's done.